internal/acctest/acctest.go

```diff
@@ -135,6 +136,10 @@ const (
 	DefaultRandomSuffixLength = 10
 )
 
+func RandStr() string {
+	return acctest.RandStringFromCharSet(DefaultRandomSuffixLength, acctest.CharSetAlphaNum)
+}
+
```

Comment:
Could reuse this in a few places:

* https://github.com/aiven/terraform-provider-aiven/blob/89689cce435c444126e6a6832dd32d109bbb6dbd/internal/plugin/service/organization/organization_application_user_test.go#L29
* https://github.com/aiven/terraform-provider-aiven/blob/75cde0c24279e482d3c1e45b69e9fd236ae90592/internal/plugin/service/organization/organization_group_project_test.go#L23
* https://github.com/aiven/terraform-provider-aiven/blob/b8dd02d4a97401a085c0e8a69265d7d278487033/internal/plugin/service/organization/organization_test.go#L22
* https://github.com/aiven/terraform-provider-aiven/blob/89689cce435c444126e6a6832dd32d109bbb6dbd/internal/plugin/service/organization/organization_user_group_member_test.go#L25

Reply:
As a separate refactoring PR, yes. When we will have the capacity to do it.